app/src/main/java/net/squanchy/service/firebase/FirebaseSquanchyRepository.java

```diff
@@ -71,12 +72,32 @@ public FirebaseSquanchyRepository(DatabaseReference database) {
         return observeChild("sessions", FirebaseEvent.Holder.class);
     }
 
+    public Observable<FirebaseEvent> event(int dayId, int eventId) {
```

Comment:
You can't drill down to the children from there, I think. Maybe. Don't remember what late-night, whisky-sipping me thought, really.